playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: controller 0 hosts file
+  hosts: controller[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[0] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[0] }}"
+
+- name: controller 1 hosts file
+  hosts: controller[1]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add controller node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[1] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[1] }}"
+
+- name: compute 0 hosts file
+  hosts: compute[0]
+  vars_files:
+  - ../vars/main.yml
+  tasks:
+    - name: add compute node to /etc/hosts
+      lineinfile: dest=/etc/hosts regexp={{ testenv_instance_names[2] }}
+                  insertafter=EOF
+                  line="127.0.1.1 {{ testenv_instance_names[2] }}"
+
+- name: umount ephemeral disk on ceph_osds
+  hosts: ceph_osds
+  tasks:
+    - name: umount ephemeral disk on ceph_osds
+      shell: umount /dev/{{ item }}
+      with_items: "{{ ceph.disks }}"
+
+- name: tasks for all
+  hosts: all
+  serial: 10
```

Comment:
Why serial? And you're doing serial 10, which is more than the number of nodes anyway.